src/components/SkillPanel.tsx

```diff
@@ -35,7 +37,7 @@ const SkillPanel = ({
         : agency.join("")
       : undefined;
   const link = new URL(
-    `${NEXT_PUBLIC_BASE_URL}/skills/${context}/${category}/${skill.name}`
```

Comment:
I think the `category` can be removed from `const { category, agency } = query;` since it's not being used anymore